webserver/run.sh

```diff
@@ -1,9 +1,35 @@
 #!/bin/bash
+# ./webserver/run.sh --REBUILD rebuild --DB mysql --DBVERSION 5.6
+DB=${DB:-mysql}
+DBVERSION=${DBVERSION:-5.6}
+REBUILD=${REBUILD:-}
 
-# Create the www directory as the current user so all subdirs will inherit the permissions
+# https://brianchildress.co/named-parameters-in-bash/
+while [ $# -gt 0 ]; do
+   if [[ $1 == *"--"* ]]; then
+        param="${1/--/}"
+        declare $param="$2"
+   fi
+  shift
+done
+
+if [[ $DB == "mysql" && $DBVERSION == "5.6" ]]; then
+  echo 'Everything is fine. We use mysql 5.6. Please delete the mysql_data directory manually if the mysql version has changed and you run into an error.'
+elif [[ $DB == "mysql" && $DBVERSION == "8" ]]; then
+  echo 'Everything is fine. We use mysql 8. Please delete the mysql_data directory manually if the mysql version has changed and you run into an error.'
+elif [[ $DB == "postgres" ]]; then
+  echo 'Everything is fine. We use the latest version of postgres. If you entered a version number, it will be ignored.'
```

Comment:
Postgres also supports different versions, so we can use the $DBVERSION parameter there as well. Or do you see any issue?